qemu/tests/block_resize.py

```diff
@@ -158,6 +158,9 @@ def get_md5_of_file(filename):
             error_context.context("Extend disk to %s in guest"
                                   % block_size, logging.info)
             if os_type == 'windows':
+                max_block_size = int(params["max_block_size"])
+                if int(block_size) >= max_block_size:
+                    test.cancel("cancel the block resize test for more than %dB disk." % max_block_size)
```

Comment:
The description can be more clear.
How about "Cancel the test if the disk size is greater than the maximum"

Reply:
have changed  the description thanks.